infrastructure/lambda-create-doc-ref.tf

```diff
@@ -92,7 +92,11 @@ module "create-doc-ref-lambda" {
   depends_on = [
     aws_api_gateway_rest_api.ndr_doc_store_api,
     module.document_reference_dynamodb_table,
+    module.ndr-lloyd-george-store,
     module.create-doc-ref-gateway,
-    module.ndr-app-config
+    module.ndr-app-config,
+    module.lloyd_george_reference_dynamodb_table,
+    module.ndr-document-store,
+    module.document_reference_dynamodb_table
```

Comment:
Scott removed the buckets on his ticket, So can you remove it here so we don't re-add them here?